src/05-invoicing/invoices.apib

```diff
@@ -144,7 +148,9 @@ Get details for a single invoice.
                                     + Members
                                         + percentage
                                 + value: 15.00 (number)
+                                + absolute_value (Money)
```

Comment:
PP : I would also call this "discount", because it is the discount you get...
Q : I people at some point are able to provide a unit price that is tax inclusive, will this value be also tax inclusive in that case? should we specify like we do with the unit price? or do we need to put tax inclusivity/exclusivity on the line level?